content/analyst/inbox-and-design-tracker/setting-persistent-campaign-filters.md

```diff
@@ -0,0 +1,40 @@
+---
+lastUpdated: "02/23/2023"
+title: Setting Persistent Campaign Filters
+description: Persistent Campaign Filters and Scheduled Analytics Exports
+---
+
+
+ This Inbox Tracker feature is designed to improve your experience and make it easier to see a targeted set of campaign data.
+
+## Persistent Campaign Tracker Filters
+
+ Within Campaign Tracker, we have expanded the filter options and given you the ability to make your filters persistent. You can now create or update profiles to focus on a specific set of campaign results beyond the sending domain.
+
+ The available campaign filters are:
+
+ -- Subject Line Keyword
+
+ -- Campaign ID
+
+ -- From Address
+
+ -- Emotion
+
+ -- Deliverability (Inbox or Spam Percentage) *(New)* 
+
+ -- Target Country *(New)* 
+
+ -- Custom Header Key *(New)* 
```

Comment:
This isn't that important but I think it looks bad to have the spaces between these (could also do a markdown bullet list)